Examples/GraphqlCall/Program.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Indico;
+using Indico.Request;
+using Newtonsoft.Json.Linq;
+
+namespace Examples
+{
+    /// <summary>
+    /// Example for raw GraphQL call. Uses Indico V1 client.
+    /// </summary>
+    internal class GraphQLCall
+    {
+        private static string GetTokenPath() =>
+            Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile),
+                "indico_api_token.txt");
+
+        public static async Task Main()
+        {
+            var config = new IndicoConfig(GetTokenPath(), "app.indico.io");
```

Comment:
Ideally we could wrap these in a V2 call/interface like the other straggler V1 calls, so we can rewrite it in the future without also changing the interface.